nncf/common/compression.py

```diff
@@ -103,7 +103,7 @@ def export_model(self,
                 - ({'x': None, 'y': y},) for keyword arguments only.
         """
         self.prepare_for_export()
-        backend = infer_backend_from_model(self.model)
```

Comment:
Why the renaming?

Reply:
My proposal, this is because the word `infer` is reserved for other things in DL.